arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,578 @@
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "summary": "Lists all of the available Automation REST API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Automation/automationAccounts": {
+      "get": {
+        "tags": [
+          "AutomationAccounts"
+        ],
+        "summary": "Lists the Automation Accounts within an Azure subscitption.",
+        "operationId": "AutomationAccounts_ListBySubscriptionId",
+		"description": "Retrieve a list of accounts within a given subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AutomationAccountListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the sku name of the account.",
+          "enum": [
+            "Free",
+            "Basic"
+          ],
+          "x-ms-enum": {
+            "name": "SkuNameEnum",
+            "modelAsString": true
+          }
+        },
+        "family": {
+          "type": "string",
+          "description": "Gets or sets the sku family."
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the sku capcity."
+        }
+      },
+      "description": "The account sku."
+    },
+	"AutomationAccountUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets account sku."
+        }
+      },
+      "description": "The parameters supplied to the patch account properties."
+    },
+    "AutomationAccountUpdateParameters": {
```

Comment:
Is this change from Hyak? For Hyak it was requirement to have Gets or Sets in description so that user knows if it is only gettable and settable property. Here is the MSDN documentation for released SDK

https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.automation.models.automationaccountcreateorupdateproperties

Since this swagger is generation from Hyak we have gets or sets in description.